### PR TITLE
[SKIP CI] Update the vDVS documentation

### PR DESCRIPTION
**Pull request by ashahi1**
Addresses the issue #2003

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsp
    - Needs [open vm tools or VMware Tools installed](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=340) ```sudo apt-get install open-vm-tools```
 - RedHat and CentOS
 - Windows Server 2016 (64 bit)
-- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later)
+- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later) [Photon 2.0] (https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)
```
Comment (shuklanirdesh82) [inline README.md]: `(v4.4.51 or later) [Photon 2.0] (https` => `(v4.4.51 or later), [Photon 2.0](https`

1. need separator between two versions
2. no need of space to show hyperlink correctly

**Comment by ashahi1** [comment]
@shuklanirdesh82 I have addressed your comments. Can you please take a look at it again?

Thanks!

File: README.md [hunk 2]
```diff
@@ -95,7 +95,8 @@ Please refer [vDVS configuration page](http://vmware.github.io/docker-volume-vsp
    - Needs [open vm tools or VMware Tools installed](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=340) ```sudo apt-get install open-vm-tools```
 - RedHat and CentOS
 - Windows Server 2016 (64 bit)
-- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later) [Photon 2.0] (https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)
+- [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later)
+- [Photon 2.0](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)
```
Comment (shuklanirdesh82) [inline README.md]: it should be in the same line ... either you may add [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later) and [above](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries) or in the same line [Photon 1.0, Revision 2](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-10-revision-2-binaries) (v4.4.51 or later), [Photon 2.0](https://github.com/vmware/photon/wiki/Downloading-Photon-OS#photon-os-20-ga-binaries)

**Comment by shuklanirdesh82** [comment]
@ashahi1 following comment (mentioned at https://github.com/vmware/docker-volume-vsphere/pull/2009#pullrequestreview-80401667) is still left to address.

Can you please correct the information for http://vmware.github.io/docker-volume-vsphere/documentation/faq.html#how-to-install-and-use-the-driver?

**Pull request merged**